crate/cli/src/actions/shared/wrap_key.rs

```diff
@@ -80,11 +79,9 @@ impl WrapKeyAction {
                 .with_context(|| "failed decoding the wrap key")?;
             create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
         } else if let Some(password) = &self.wrap_password {
-            let key_bytes = derive_key_from_password::<SYMMETRIC_WRAPPING_KEY_SIZE>(
-                password.as_bytes(),
-                KMS_ARGON2_SALT,
-            )?
-            .to_vec();
+            let key_bytes =
+                derive_key_from_password::<SYMMETRIC_WRAPPING_KEY_SIZE>(password.as_bytes())?
+                    .to_vec();
             create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
```

Comment:
I don't think you need the `to_vec()`.
```suggestion
            let key_bytes =
                derive_key_from_password::<SYMMETRIC_WRAPPING_KEY_SIZE>(password.as_bytes())?;
            create_symmetric_key(&key_bytes, CryptographicAlgorithm::AES)
```